scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -680,6 +692,10 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
   private def countIndent(tokenPosition: Int): Int =
     countIndentAndNewlineIndex(tokenPosition)._1
 
+  private def currentIndentation: Int = {
```

Comment:
unrelated question to method 10 lines above. why does it check for whitespace in two differently ways? isWhitespace vs is[Whitespace].

Reply:
`isWhitespace` is only used to count spaces or tabs, so in reality it should be `isSpacesOrTabs`, but it's an inner function, so this should not be a big issue.